src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private double length;
+    private double height;
+
+    public Rectangle(String color, double length, double height) {
+        super(color);
+        this.length = length;
+        this.height = height;
+    }
+
+    public double calculateArea() {
```

Comment:
You should not use verbs for method names within a class that represents an entity. Consider renaming `calculateArea` to `getArea` to align with the convention that getters typically start with 'get' and represent properties of the object.